test.js

```diff
@@ -76,6 +76,21 @@ resetMock({status: 0, stdout: 'null'})
 result = dockerLambda({returnSpawnResult: true})
 result.should.eql({status: 0, stdout: 'null'})
 
+// Should not fail if stdout contains logging
+resetMock({status: 0, stdout: 'Test\nResult\n{"success":true}'})
+result = dockerLambda()
+result.should.eql({success: true})
+
+// Should return undefined if last stdout entry cannot be parsed
+resetMock({status: 0, stdout: 'Test\nResult\nsuccess'})
+result = dockerLambda()
+result === void 0 // Result is undefined and does not contain `should`
```

Comment:
You'll need to assert something here. Just `should.not.exist(result)` would be fine 👍